modules/src/twitter.py

```diff
@@ -0,0 +1,56 @@
+import os
+import config
+import urllib
+
+from birdy.twitter import UserClient
+from templates.text import *
+
+TWITTER_CONSUMER_KEY = os.environ.get('TWITTER_CONSUMER_KEY', config.TWITTER_CONSUMER_KEY)
+TWITTER_CONSUMER_SECRET = os.environ.get('TWITTER_CONSUMER_SECRET', config.TWITTER_CONSUMER_SECRET)
+TWITTER_ACCESS_TOKEN = os.environ.get('TWITTER_ACCESS_TOKEN', config.TWITTER_ACCESS_TOKEN)
+TWITTER_ACCESS_TOKEN_SECRET = os.environ.get('TWITTER_ACCESS_TOKEN_SECRET', config.TWITTER_ACCESS_TOKEN_SECRET)
+
+CHARACTER_LIMIT = 1024
+
+
+def process(input, entities=None):
+    keyword = None
+    if entities is not None and len(entities) > 0:
+        keyword = entities[0]["value"]
```

Comment:
`len(entities) > 0` seems to be a sufficient condition to test `entities is not None`. You can also join the condition. `keyword = entities[0]["value"] or "news"`